include/ismrmrd/ismrmrd.h

```diff
@@ -88,16 +88,31 @@ enum TextType {
   ISMRMRD_XML_HEADER_TEXT = 1
 };
 
+// !! If changing StreamID enumeration - make sure to update the streamIdToString function in ismrmrd.cpp !!
+// TODO Most of the values bellow are just for testing
 enum StreamId {
-  ISMRMRD_STREAM_INVALID        =  0,
-  ISMRMRD_HEADER_STREAM         =  1,
-  ISMRMRD_HANDSHAKE_STREAM      =  2,
-  ISMRMRD_COMMAND_STREAM        =  3,
-  ISMRMRD_ERROR_STREAM          =  4,
-  ISMRMRD_MRACQUISITION_STREAM  =  5,
-  ISMRMRD_IMAGE_STREAM          =  6,
-  ISMRMRD_WAVEFORM_STREAM       =  7,
-  ISMRMRD_BLOB_STREAM           =  8
+  STREAM_NONE                   =    0,
+  STREAM_HEADER                 =    1,
+  STREAM_HANDSHAKE              =    2,
+  STREAM_COMMAND                =    3,
+  STREAM_ERROR                  =    4,
+  STREAM_MRACQUISITION_DEFAULT  =  100,
```

Comment:
Why not just have `STREAM_MRACQUISITION_1` be the default or better yet `STREAM_MRACQUISITION_0`